test/app/core/extension/safely_parser_test.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:penhas/app/core/extension/safely_parser.dart';
+
+void main() {
+  group('SafelyParser extension', () {
+    test('safeParseDouble should parse double from string or number', () {
+      expect((1.0).safeParseDouble(), 1.0);
+      expect(('1.0').safeParseDouble(), 1.0);
+      expect(('not a number').safeParseDouble(), isNull);
+      expect((null).safeParseDouble(), isNull);
+    });
+
+    test('safeParseInt should parse int from string or number', () {
+      expect((1).safeParseInt(), 1);
+      expect(('1').safeParseInt(), 1);
+      expect(('not an int').safeParseInt(), 0);
+      expect((null).safeParseInt(), 0);
```

Comment:
Acha que seria uma boa fazer uma validação se está retornando o valor `def` quando o valor não for um numérico?